lib/perl/Genome/VariantReporting/Framework/Component/Report/MergeCompatible.pm

```diff
@@ -23,4 +23,10 @@ sub report_path {
         $self->class);
 }
 
+sub has_size {
+    my $self = shift;
+
+    return -s $self->report_path;
+}
```

Comment:
Possibly. Although I though we should try to avoid memoizing with/on an object. I might just makes this a transient attribute.